rust-src/concordium_base/src/web3id/mod.rs

```diff
@@ -1,11 +1,12 @@
-//! Functionality related to constructing and verifying Web3ID proofs.
+//! Functionality related to constructing and verifying V0 Concordium Web3ID proofs.
```

Comment:
`V0` (are these the old `web3id` proofs) as this module also includes the `V1` flow?